CMakeLists.txt

```diff
@@ -497,6 +497,10 @@ if(FIPS AND (NOT GO_EXECUTABLE OR NOT PERL_EXECUTABLE))
   message(FATAL_ERROR "Building AWS-LC for FIPS requires Go and Perl")
 endif()
 
+if(FIPS AND NOT BUILD_SHARED_LIBS)
+  message(FATAL_ERROR "Building AWS-LC for FIPS only supports the shared build, you must pass in '-DBUILD_SHARED_LIBS=TRUE'")
+endif()
```

Comment:
* FIPS shared build will be support soon, right? e.g. Android.
* Do we plan to just make this diff to `fips-2021-10-20` branch?

Reply:
* Yes we need to add FIPS static support back soon. When we get to doing that work it will be very obvious we need to remove this change because there will be no way to test that change with this in place.
* I was not planning on moving this to the `fips-2021-10-20` branch. For that people should be following the build instructions in our security policy. I was treating this change as a nice to have sanity test for developers developing on `main`.